docs/how-to.md

```diff
@@ -245,6 +245,50 @@ end
 !!!
 +++
 
+
+### How to Paginate While Maintaining Independent Contexts
+
+If you're using [hotwire](https://hotwired.dev/) ([turbo-rails](https://github.com/hotwired/turbo-rails) - if you're using Rails) when one way of maintaining independent contexts is as below:
+
+```html+erb
+  <-- movies#bad_movies -->
+  <%= turbo_frame_tag "bad_movies", src: bad_movies_path do %>    
+      <%= render "movies_table", locals: {movies: @movies}%>
+      <%== pagy_bootstrap_nav(@pagy) %>    
+  <% end %>
+
+  <-- movies#good_movies -->
+  <%= turbo_frame_tag "good_movies", src: good_movies_path  do %>    
+      <%= render "movies_table", locals: {movies: @movies}%>
+      <%== pagy_bootstrap_nav(@pagy) %>    
+  <% end %>   
+```
+
+```rb
+  # controller action 
+  def good    
+    @pagy, @movies = pagy(Movie.good, items: 5) 
+  end 
+
+  def bad    
+    @pagy, @movies = pagy(Movie.bad, items: 5)
+  end 
+```
+
+You may wish to scope your records using a search form (perhaps using [ransack](https://github.com/activerecord-hackery/ransack) etc.), to the same url, or a different url, or you may wish to use an entirely different model (actors, rather than movies): but the logic remains the same: wrap each independent context in a `turbo_frame_tag` and ensure each url returns a matching `turbo_frame_tag`.
```

Comment:
```suggestion
You may want to scope your records using a search form (perhaps using [ransack](https://github.com/activerecord-hackery/ransack) etc.), to the same url, or a different url, or use an entirely different model (actors, rather than movies). 

The logic remains the same: wrap each independent context in a `turbo_frame_tag` and ensure each url returns a matching `turbo_frame_tag`.
```